includes/class-smaily-for-wp-option-handler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Define the plugin's reading and writing functionality, from and to WordPress' options table.
+ *
+ * @since      3.0.0
+ * @package    Smaily_For_WP
+ * @subpackage Smaily_For_WP/includes
+ */
+class Smaily_For_WP_Option_Handler {
```

Comment:
Simply name this class `Smaily_For_WP_Options`.

Reply:
Done. 554adf3